RNAseq/RNAseq_Tool_Install/README.md

```diff
@@ -70,14 +70,14 @@
   
   4. Create all sub-directories (aka child and subsequent grandchild directories) within the top level directory using the GL_RNAseq_mkdir.sh script by executing the following command (make sure you first download the [GL_RNAseq_mkdir.sh](https://github.com/nasa/GeneLab_Data_Processing/blob/master/RNAseq/RNAseq_Tool_Install/GL_RNAseq_mkdir.sh) file and save it in the topy level direcroty you made in step 2):
   ```
-  ./GL_RNAseq_mkdir.sh
```

Comment:
also just noting i removed the `./` too because it isn't needed once we have a command in front of the filename. Just like calling `head file.txt`, `bash file.txt` just needs to be pointing to where the file is